.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,282 @@
+# How to contribute
+
+We’d love your help.
+
+* Have you noticed something that could be improved?
+* Want to share your tech writing expertise?
+* Do you have material you'd like to integrate into our baseline?
+* Do you know of research which should be referenced?
+* Something else?
+
+Please do reach out to us with your ideas.
+
+This guide covers how to get started with contributing to the Good Docs Project, including working with the project's code repository, and making your first pull request.
+It does not cover how the project handles governance issues such as decision making and community roles and responsibilities.
+For information about project governance, see [https://thegooddocsproject.dev/community.html](The Good Docs Project Community).
+
+## Welcome to The Good Docs Project
+
+The Good Docs Project provides a set of templates and other resources to help you create documentation.
+Anyone can make use of the resources, download the templates, and create their own documentation.
+This guide is to help you get started contributing to the Good Docs Project, whether you intend to contribute a whole new template to the collection, or fix a typo.
+
+## Getting Familiar with the project repositories
+
```

Comment:
**Suggestion (non-blocking):** Convert "familiar" to lowercase to match the sentence case headings used in the rest of the guide.

Reply:
Good spot! Thanks :+1: